docs/privacy.md

```diff
@@ -0,0 +1,96 @@
+
+If in your event you collect personal data and are subject to GDPR or a similar data privacy legislature,
+you may specify the name and contact information
+of your designated Data controller here.
+
+![Data controller](assets/privacy/data_controller.png)
+
+#### Privacy notice
+In the following section, you can set a privacy notice for your event.
+You may choose between linking to an external page containing the privacy notice or
+entering a privacy notice in the text field.
+
+If needed, you can also specify multiple
+external notices by clicking the *Add* button and adding a title and the external url.
+
+![Privacy notice](assets/privacy/privacy_notice.png)
+
+Regardless of whether you include an external notice or enter one manually, a link to the notice will be shown
+in the footer of the event page.
+
+
+### Participant visibility
+By default, Indico displays all registered participants on the event page.
+However, some participants may prefer that this information not be visible to everyone.
+
+To this end, Indico allows you to set the participant visibility settings for every registration form in your event.
+The visibility settings can be adjusted separately for registered participants and the general public.
+
+The visibility options are:
+
+- Hidden
+
+- Visible if the participant gives consent
+
+- Always visible regardless of consent
+
+When registering, participants are presented with an option to agree to being shown in the participant list. This corresponds to the second option from the list above.
+
+The last option which makes the participant always visible is discouraged and should not be used unless there is a legitimate interest to have the participant list visible. Moreover, this option can only be selected as long as there are no registered participants.
+
+The last section of the privacy dashboard shows the current visibility settings for all registration forms in the given event.
+
+![Data controller](assets/privacy/participant_visibility.png)
+
+The visibility settings can be managed by clicking on the cog icon corresponding to the registration form
+you wish to update. This opens the registration privacy settings which can also be reached by going to the registration form settings and selecting _Privacy settings_.
+
+![Data controller](assets/privacy/privacy_settings.png)
+
+On this page you can change the participant visibility settings and the visibility duration, which when set
+will automatically hide the participant list after a given number of months. 
+
+![Data controller](assets/privacy/visibility_settings.png)
+
+
+### Data retention period
+Indico allows you to automatically delete registration data after a set amount of time by setting a retention period.
+
+A retention period can be set registration-wide which then applies to the whole registration form. Once this period expires, 
+all registrations are automatically deleted along with any data.
```

Comment:
Maybe mention that the participant list entries will disappear as well in that case?